packages/fluentui/CHANGELOG.md

```diff
@@ -25,6 +25,8 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Rename `indicatorBorderColorDefault` variable to `indicatorColorDefault` @assuncaocharles ([#14895](https://github.com/microsoft/fluentui/pull/14895))
 - Remove the `disabled` property from `Dialog` and `Popup` behaviors @rymeskar ([#14885](https://github.com/microsoft/fluentui/pull/14885))
 - Remove `selectableParent` prop in favor of `selectable` in `TreeItem` and make `selectionIndicator` visible only on focus or hover @assuncaocharles ([#15133](https://github.com/microsoft/fluentui/pull/15133))
+- Styles from `Avatar` slots were moved to exact components (`AvatarStatus`, `AvatarStatusIcon`) @assuncaocharles ([#16382](https://github.com/microsoft/fluentui/pull/16382))
```

Comment:
I don't think it would be right to block your PR on this, but now that we have decided to avoid breaking changes as much as possible and make future upgrades easier, we need a better way of documenting deprecation/adoption plans - might bring it up at the next open hours.

Essentially we are deprecating the use of the status slot in componentStyles and removing all variables related to the styling of the slot

Reply:
To follow-up on Ling's comment - would it be possible to have code-mods for the consumers upgrade?